main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1378,20 +1359,37 @@ private void updateFavPointBox() {
         /**
          * shows/hides/updates list box
          */
-        private void updateListBox() {
-            final View box = view.findViewById(R.id.list_box);
+        public void updateListBox() {
+            final ListView listListView = (ListView) view.findViewById(R.id.list_list);
 
             if (cache.isOffline()) {
-                // show box
-                box.setVisibility(View.VISIBLE);
+                // show listListView
+                listListView.setVisibility(View.VISIBLE);
+                final ListListAdapter adapter = new ListListAdapter(CacheDetailActivity.this, cache, this);
+                listListView.setAdapter(adapter);
+
+                final int numberOfItems = adapter.getCount();
+
+                // Get total height of all items.
+                int totalItemsHeight = 0;
+                for (int itemPos = 0; itemPos < numberOfItems; itemPos++) {
+                    final View item = adapter.getView(itemPos, null, listListView);
+                    item.measure(0, 0);
+                    totalItemsHeight += item.getMeasuredHeight();
+                }
+
+                // Get total height of all item dividers.
+                final int totalDividersHeight = listListView.getDividerHeight() *
+                        (numberOfItems - 1);
 
-                // update text
-                final TextView text = ButterKnife.findById(view, R.id.list_text);
-                final StoredList list = DataStore.getList(cache.getListId());
-                text.setText(res.getString(R.string.cache_list_text) + " " + list.title);
+                // Set list height.
+                final ViewGroup.LayoutParams params = listListView.getLayoutParams();
+                params.height = totalItemsHeight + totalDividersHeight;
```

Comment:
I got a problem when adding lists to the listview. They didn't appear because the height didn't recalculate. So I searched and found this solution. I'll look for another solution. Any help/hint would be appreciated.